mava/logger.py

```diff
@@ -115,7 +115,7 @@ def log(
     return log
 
 
-def logger_setup(_run: Run, config: Dict, _log: SacredLogger):
+def logger_setup(_run: Run, config: Dict, _log: SacredLogger) -> Callable:
```

Comment:
Would be nice in future to define a type for this and some other methods throughout the code so it's not just `Callable`, but rather `Callable[int, float, etc]`